docs/sop.rst

```diff
@@ -0,0 +1,135 @@
+Operating Clowder
+=================
+
+Two primary aspects of operating Clowder: Operating the apps managed by Clowder and operating
+Clowder itself.
+
+Contrary to many other deployment models where apps configure themselves, an application that is
+managed by Clowder will have configuration presented to it and is expected to use it. Clowder
+governs many different aspects of an application's configuration, from defining the port it should
+listen on, to metrics, kafka topics, service hostnames, and more. In this way a common configuration
+format is presented to the application no matter the envrionment it is running in, enabling a far
+easier development experience.
+
+Operating Apps Managed by Clowder
+---------------------------------
+
+ClowdEnvironment
+++++++++++++++++
+
+**abbreviated to ``env`` in k8s**
+
+The ``ClowdEnvironment`` CRD is responsible for configuring the environment that the Clowder enabled
+apps will interact with. It is a *cluster scoped* CRD and thus must have a unique name inside the
+k8s cluster. For production environments it is usual to have only one ``ClowdEnvironment``, whereas
+in other scenarios &mdash; such as ephemeral testing &mdash; Clowder enables the management of
+multiple environments that operate completely independently from each other.
+
+Providers
+^^^^^^^^^
+
+Clowder provides a number of providers which govern the creation of services, e.g. Kafka topics,
+object storage, etc, that applications may depend on. The ``ClowdEnvironment`` CRD configures these
+providers principally by making use of a provider's mode.
+
+Modes
+^^^^^
+
+Providers often operate in different modes. As an example the Kafka provider can operate in three
```

Comment:
This first sentence seems redundant with the last sentence of the previous paragraph.  Maybe remove that last sentence??

Reply:
Sure - but if a section is navigated to having a brief context isn't a bad idea.